apps/modernization-ui/src/pages/patient/profile/sexBirth/SexBirthForm.tsx

```diff
@@ -87,7 +87,9 @@ export const SexBirthForm = ({ entry, onChanged, onCancel }: Props) => {
         <>
```

Comment:
@adamloup-enquizit  Added Label for each input. Should this be down in all files or should I remove the labels and bold without the labels. Want to do the most correct thing.